source/Halibut.Tests/WhenCallingAServiceThatDoesNotExist.cs

```diff
@@ -20,10 +21,10 @@ public async Task AServiceNotFoundHalibutClientExceptionShouldBeRaisedByTheClien
                        .WithHalibutLoggingLevel(LogLevel.Info)
                        .Build(CancellationToken))
             {
-                var echo = clientAndService.CreateClient<IEchoService>();
-                Func<string> readAsyncCall = () => echo.SayHello("Say hello to a service that does not exist.");
+                var echo = clientAndService.CreateClient<IEchoService, IAsyncClientEchoService>();
```

Comment:
Did you want to leave this testing only testing sync for now or did you miss the attribute override?

Reply:
missed it thanks